docs/guides/advanced-usage.md

```
@@ -2,10 +1,79 @@
+
+```bash
+kuberctl get pods # List all pods
+
+# Enter the running container
+kubectl exec -it <pod-name> -- /bin/bash # Enter a pod
 
-- EC2 + PostgreSQL + Nginx + Gunicorn setup
-- Use .env files for environment configuration
+# Tail logs
+kubectl logs -f deployment/django-mindjunkies-web-deployment
+```
```

markdownlint-cli2 (0.17.2)

Files should end with a single newline character
null

(MD047, single-trailing-newline)